client/my-sites/earn/memberships/index.jsx

```diff
@@ -336,7 +366,9 @@ class MembershipsSection extends Component {
 							action: 'cancel',
 						},
 						{
-							label: this.props.translate( 'Disconnect Recurring Payments from Stripe' ),
+							label: isPayments
+								? this.props.translate( 'Disconnect Payments from Stripe' )
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Disconnect Recurring Payments from your Stripe account' )` *ES Score: 7*